halyard/hal-scripts/setup-ingress-content.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+if [ ! -d /${USER}/ingress ]; then
+  mkdir /${USER}/ingress
+fi
+
+%{ for deployment, details in deployments ~}
+
+echo "Creating Tiller service account and RBAC for deployment ${deployment}"
+
+cat << TILLER_RBAC | kubectl --kubeconfig="${details.kubeConfig}" create -f -
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tiller
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tiller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+
+TILLER_RBAC
+
+echo "Initializing Helm for deployment ${deployment}"
```

Comment:
This would take some testing as the vault-helm site says `Important Note: This chart is not compatible with Helm 3. Please use Helm 2 with this chart.`

https://www.vaultproject.io/docs/platform/k8s/helm/

Reply:
this should be it's own pr. added an issue. 
https://github.com/homedepot/spingo/issues/107